docs/transpile/create-a-transpiler-plugin.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>Thanks! I think it is helpful to have the syntax for all the kinds of entry points here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>